tests/nix/suite/utils.nix

```
@@ -0,0 +1,26 @@
+{ testName }:
+let
+  flake = import ../../../.;
+in
+{
+  popTest = system: name: cfg: {
```

Maybe rename this to something more descriptive like

```suggestion
  mkHiveNode = system: name: cfg: {
```